src/gobimportclient/import_client.py

```diff
@@ -67,6 +90,8 @@ def read(self):
         else:
             raise NotImplementedError
 
+        self.log('info', f"Data has been imported from {self.source['name']}.")
```

Comment:
Looks if only the happy flow is logged?

Reply:
Error flow is logged by catching exceptions raised in connect, read and validate.

Reply:
OK